_security-analytics/sec-analytics-config/correlation-config.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Creating correlation rules
+parent: Setting up Security Analytics
+nav_order: 16
+---
+
+# Creating correlation rules
+
+The correlation engine is an experimental feature released in OpenSearch 2.7. Therefore, we do not recommend using the feature in a production environment at this time. For updates on the progress of correlation engine, see [Security Analytics Correlation Engine](https://github.com/opensearch-project/security-analytics/issues/369) at GitHub. To share ideas and provide feedback, join the [Security Analytics forum](https://forum.opensearch.org/c/plugins/security-analytics/73).
+{: .warning }
+
+Correlation rules allow you to define threat scenarios involving multiple systems in an infrastructure by matching the signatures of threat events occuring in different log types. Once a rule contains at least two different log sources and the preferred fields and field values that define an intended threat secenario, the correlation engine can query the indexes specified in the correlation rule and draw any correlations found between the findings.
```

Comment:
In the last sentence, is "draw" the right verb? If so, would "and draw any correlations between the findings" (remove "found") work?